src/sass/partials/_sidebar.scss

```diff
@@ -0,0 +1,20 @@
+aside.pf-c-page__sidebar {
+    padding-top: 0px;
+
+    .ins-c-navigation__header {
+        max-height: 4.4rem;
```

Comment:
According to the mocks, this should be the same height as the page header. Also, we need to use the rem utility here

Reply:
Good point!